src/acceptance-test/java/org/zalando/nakadi/webservice/hila/HilaAT.java

```diff
@@ -207,7 +211,7 @@ public void whenConnectionIsClosedByClientNakadiRecognizesIt() throws Exception
         waitFor(() -> assertThat(client.getBatches(), hasSize(1)));
 
         client.close();
-        Thread.sleep(2000);
+        Thread.sleep(10000);
```

Comment:
why it was increased so significantly? As I understand in this PR you just added new property to the batch, I expected that it will not increase any timeouts etc.